ci_scripts/ubuntu-19_04/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:19.04
+
+ENV BUILDER_NAME="ubuntu-19_04"
+
+# Update system
+RUN true \
+    && apt update \
+    && apt -y upgrade
+
+# Install generic dependencies
+RUN apt -y install curl
+
+# Install nvm & node
+COPY ./install-nvm-node.sh /tmp
+RUN chmod +x ./tmp/install-nvm-node.sh && /tmp/install-nvm-node.sh
+
+# Install dependencies for deltachat-node
+RUN apt -y install python2
+
+# Install rust
+COPY ./install-rust.sh /tmp
+RUN chmod +x ./tmp/install-rust.sh && /tmp/install-rust.sh
+ENV PATH=/root/.cargo/bin:$PATH
+
+# Install dependencies for electron builder
+RUN apt -y install binutils
```

Comment:
Do we really need any of these docker files? It should be enough to `npm install` and run `electron-builder`.

Reply:
It's possible that we don't need them anymore, i would maybe keep one to have a more or less stable environment to build the .deb . Not sure if i want to build an ubuntu package on a arch distro.

Reply:
Go for it!